.github/workflows/benchmark.yml

```diff
@@ -3,6 +3,7 @@ name: Benchmark
 on:
   issue_comment:
     types: [created]
+  workflow_dispatch:
```

Comment:
Question: Some of the workflows (like this one) rely on the `github.event` variable to work. IIUC this workflow relies on specific properties to the `issue_comment` event only, does it work for `workflow_dispatch` too?